### PR TITLE
removed the filed data_type

### PR DESCRIPTION
**Pull request by ganqqwerty**

### CODE REVIEW

**Comment by lomigmegard** [comment]
Should you also add the new schema version, or we keep the same RC?

**Comment by ganqqwerty** [comment]
@tarfaa can you check? we have support for the new fields in the protocol now

**Comment by cathzwah** [comment]
"publication": {
     "type": "string",
    "description": "The doi of the publication where the protocol is described" }

REPLACE by

""publication_doi": {
                            "type": "string",
                            "description": "The doi of the publication where the protocol is described"
                          }
"publication_pubmed": {
                         "type": "string",
                           "description": "The PubMed of the publication where the protocol is described"
          }

File: neuroinformatics/single_activity_metadata/single-activity-metadata-schema-1.0.json [hunk 1]
```diff
@@ -25,6 +25,22 @@
                     "type": "string",
                     "format": "date"
                 },
+                "agreement": {
+                    "description": "Information related to user accepting terms and conditions.",
+                    "type": "object",
+                    "required": ["timestamp", "version"]
```
Comment (samuel-kerrien) [inline neuroinformatics/single_activity_metadata/single-activity-metadata-schema-1.0.json]: comma missing

File: neuroinformatics/single_activity_metadata/single-activity-metadata-schema-1.0.json [hunk 2]
```diff
@@ -25,6 +25,22 @@
...
+                    "properties": {
+                        "timestamp": {
+                            "description": "Time by which the user agreed with the terms and conditions in the epoch format ",
```
Comment (samuel-kerrien) [inline neuroinformatics/single_activity_metadata/single-activity-metadata-schema-1.0.json]: add "in milliseconds" to make it clear.

File: neuroinformatics/single_activity_metadata/single-activity-metadata-schema-1.0.json [hunk 3]
```diff
@@ -25,6 +25,22 @@
...
+                            "type": "number"
+                        },
+                        "version": {
+                            "description": "Version of TOC user agreed with.",
```
Comment (samuel-kerrien) [inline neuroinformatics/single_activity_metadata/single-activity-metadata-schema-1.0.json]: Write it "terms and conditions" as you have done twice already above

**Pull request merged**